.pluginsync.yml

```diff
@@ -0,0 +1,18 @@
+# File managed by pluginsync
+pluginsync_config: '0.1.8'
+managed_files:
+- .github
+- .github/ISSUE_TEMPLATE.md
+- .github/PULL_REQUEST_TEMPLATE.md
+- .gitignore
+- .pluginsync.yml
+- .travis.yml
+- CONTRIBUTING.md
+- LICENSE
+- Makefile
+- scripts
+- scripts/build.sh
+- scripts/common.sh
+- scripts/deps.sh
+- scripts/pre_deploy.sh
+- scripts/test.sh
```

Comment:
Yep, looks like the file listing in .pluginsync.yml didn't scan for unmanaged files.